lib/settings/scripts/ethiopic/_config.scss

```diff
@@ -0,0 +1,3 @@
+@if $script == 'ethiopic' {
+
```

Comment:
Just to note this is empty as the ethopic script is yet to have a typography review but we are expecting this to be filled with overrides.